src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4178,8 +4201,10 @@ namespace AirflowNetworkBalanceManager {
                     }
                     if (!(SurfWinOriginalClass(MultizoneSurfaceData(count).SurfNum) == SurfaceClass_Window ||
                           SurfWinOriginalClass(MultizoneSurfaceData(count).SurfNum) == SurfaceClass_GlassDoor ||
-                          SurfWinOriginalClass(MultizoneSurfaceData(count).SurfNum) == SurfaceClass_Door)) {
-                        ShowSevereError(RoutineName + "AirflowNetworkComponent: The opening must be assigned to a window, door or glassdoor at " +
+                          SurfWinOriginalClass(MultizoneSurfaceData(count).SurfNum) == SurfaceClass_Door ||
+                          Surface(MultizoneSurfaceData(count).SurfNum).IsAirBoundarySurf)) {
```

Comment:
Allow openings to be attached to an air boundary surface, even if it's a base surface (not a door or window). Here and below for other opening types.

Reply:
We defined a wall as a surface, but not a subsurface. We define a window as a subsurface, but not a surface. Do we allow an AirBoundarySurf as a surface and a subsurface? I prefer to define AirBoundarySurf either a surface or a subsurface, not both for clarity.

Reply:
Construction:AirBoundary can be applied to both a base surface and a fenestration subsurface. I agree this causes some confusion in the code (and for users). That may change in the future, but for now, that's how it it.